src/wp-includes/block-template-utils.php

```diff
@@ -696,7 +696,12 @@ function get_block_templates( $query = array(), $template_type = 'wp_template' )
 
 			$is_not_custom   = false === array_search(
 				wp_get_theme()->get_stylesheet() . '//' . $template_file['slug'],
-				array_column( $query_result, 'id' ),
+				array_map(
+					function ( $template ) {
+						return $template->id;
+					},
+					$query_result
+				),
```

Comment:
```suggestion
				wp_list_pluck( $query_result, 'id' ),
```
[wp_list_pluck](https://developer.wordpress.org/reference/functions/wp_list_pluck/) appears to function as expected in PHP 5.6 and some testing with `microtime()` appears to show `wp_list_pluck()` outperforming `array_map()` in this instance.

Reply:
yes it looks good to me to use `wp_list_pluck()` instead of `array_map()`

Reply:
Updated to use `wp_list_pluck()` (I wasn't aware it was a thing, but I do now!)

Confirmed it works on our PHP 5.6 development environment

All tests are passing

Reply:
@costdev / @rudlinkon - GitHub won't let me re-request a review, but the changes are now made, and tests passing 👍